platform-includes/performance/configure-sample-rate/javascript.cloudflare.mdx

```diff
@@ -0,0 +1,65 @@
+<SignInNote />
+
+```javascript
+import * as Sentry from "@sentry/bun";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [],
+});
+```
+
+## Cloudflare Pages
+
+To use this SDK, add the `sentryPagesPlugin` as
+[middleware to your Cloudflare Pages application](https://developers.cloudflare.com/pages/functions/middleware/).
+
+We recommend adding a `functions/_middleware.js` for the middleware setup so that Sentry is initialized for your entire
+app.
+
+```javascript
+// functions/_middleware.js
+import * as Sentry from "@sentry/cloudflare";
+
+export const onRequest = [
+  // Make sure Sentry is the first middleware
+  Sentry.sentryPagesPlugin((context) => ({
+    dsn: "___PUBLIC_DSN___",
+    // We recommend adjusting this value in production, or using tracesSampler
+    // for finer control
+    tracesSampleRate: 1.0,
+
+    // Set `tracePropagationTargets` to control for which URLs trace propagation should be enabled
+    tracePropagationTargets: ["localhost", /^https:\/\/yourserver\.io\/api/],
+  })),
+  // Add more middlewares here
+];
+```
+
+## Cloudflare Workers
+
+To use this SDK, wrap your handler with the `withSentry` function. This will initialize the SDK and hook into the
+environment. Note that you can turn off almost all side effects using the respective options.
+
+Currently only ESM handlers are supported.
```

Comment:
l: Same as above: I'd pack this into a `<Note>` and move it to after the snippet, wdyt?